src/DeploymentEnvironments.ts

```diff
@@ -137,6 +137,22 @@ export interface MonitoringRule {
      */
     excludeEvents?: string[];
   };
+
+  /**
+   * Configuration for monitoring deployments
+   */
+  deployMonitoring?: {
```

Comment:
Ietwat verwarrende naam gezien hoe het project heet :), `monitorDeployments` is voor mij al logischer om een of andere reden.

Reply:
Goed punt, wat denk je van localDeployMonitoring? Ook al kan het in principe ook wat anders monitoren als je de properties wat aanpast